src/gfn/gym/helpers/box_utils.py

```diff
@@ -35,29 +34,28 @@ def __init__(
         self,
         delta: float,
         northeastern: bool,
-        centers: TT["n_states", 2],
-        mixture_logits: TT["n_states", "n_components"],
-        alpha: TT["n_states", "n_components"],
-        beta: TT["n_states", "n_components"],
+        centers: States,  # TODO: should probably be a tensor
```

Comment:
This was typed as Tensor, but it is a States. It makes more sense to be a Tensor, but for simplicity, I didn't change it in this PR.